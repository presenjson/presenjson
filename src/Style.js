import React from 'react';
import PropTypes from 'prop-types';
import cx from 'classnames';

const Style = ({ children, style, ...rest }) => (
    <div className={cx(rest, 'container')} style={style}>
        {children}
    </div>
);

Style.propTypes = {
    children: PropTypes.node,
    style: PropTypes.object
};

export default Style;
