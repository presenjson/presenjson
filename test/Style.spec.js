import React from 'react';
import { shallow } from 'enzyme';
import Style from '../src/Style';

describe('Style', () => {
    it('renders correctly', () => {
        expect(
            shallow(
                <Style blur fullscreen>
                    <div className="test" />
                </Style>
            )
        ).toMatchSnapshot();
    });
});
