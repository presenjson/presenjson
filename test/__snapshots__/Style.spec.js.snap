// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Style renders correctly 1`] = `
<div
  className="blur fullscreen container"
>
  <div
    className="test"
  />
</div>
`;
